roles/tendrl-storage-node/tasks/firewalld.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Install firewalld
```

Comment:
I understand this doesn't hamper the prots enabled for gluster bricks etc. It just adds extra one needed for  webhook?

Reply:
You are right, this would break glusterfs setup if firewalld is unconfigured and disabled.

Reply:
The issue in which I suggested how to approach this (https://github.com/Tendrl/tendrl-ansible/issues/49) states:

> when firewalld service is not up and running, stop the playbook(via assert) and ask for manual intervention

Reply:
yes,In that case better we can skip firewall configuration for node and server if its already disabled. Although there is no such known issues/limitation in tendrl-server, there might be some app pkgs which are running to work in firewalld disabled mode could behave improperly. So as to avoid this, we can skip firewall configuration if its already in disabled in that machine.  
What is your view on this ?